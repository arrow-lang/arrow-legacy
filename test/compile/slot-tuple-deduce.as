// Deduce
let a = (30, 15.31, false);
