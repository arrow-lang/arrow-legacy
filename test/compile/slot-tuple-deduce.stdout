@a = constant { i32, double, i1 } { i32 30, double 1.531000e+01, i1 false }
